src/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md

```diff
@@ -15,11 +15,15 @@ Quarterly releases may contain backward-incompatible changes (BIC). Magento 2.3.
 
-With this quarterly release, we’ve changed how we describe these security issues. Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see Security updates available for Magento.
+With this quarterly release, we’ve changed how we describe these security issues. Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an [Adobe Security bulletin](https://helpx.adobe.com/security/products/magento/apsb20-22.html).
 
 ## Other release information
 
-Although code for these features is bundled with quarterly releases of the Magento core code, several of these projects (for example, Page Builder, Inventory Management, and Progressive Web Applications (PWA) Studio) are also released independently. Bug fixes for these projects are documented in the separate, project-specific release information that is available in the documentation for each project.
+Although code for these features is bundled with quarterly releases of the Magento core code, several of these projects (for example, Inventory Management and Progressive Web Applications (PWA) Studio) are also released independently. Bug fixes for these projects are documented in the separate, project-specific release information that is available in the documentation for each project.
+
+## Download and run the updated Database Cleanup script
+
+This hotfix addresses an issue with a previous database clean-up script that was released in March 2020. That database cleanup script has been updated to clear pre-existing failed login data in additional database tables. **We recommend that all merchants run DB_CLEANUP_SCRIPT_v2 script to clear pre-existing failed login data in additional tables as soon as possible**. See the [Remove failed login attempts from the database](https://support.magento.com/hc/en-us/articles/360040209352?mkt_tok=eyJpIjoiTnpCbE1UQmxOV05oT0RSbSIsInQiOiJGdlcxeG1pMlwvNTFaZXdpeFFCT2ltVHZwMjZ3Rk5pcHZubitJVW10ck1PR0xOK2dyam9DMWVWeTcxbStWbExOZ085UVg3VFk5S0szUzBTekMwYVVMWXc9PSJ9) support article.
```

Comment:
The link is not working (which was also mentioned in the #security channel on Slack by Dominik). Also, that long query string is probably unneeded, `https://support.magento.com/hc/en-us/articles/360040209352` should be enough. The same link is used in https://magento.com/security/patches/hot-fix-available-cve-2019-8118

Reply:
Link seems to work again today, but still I think that query string is unneeded.

Reply:
I shortened the link in the source -- thanks